app/views/devise/registrations/new.html.erb

```diff
@@ -29,6 +30,8 @@
   <div class="actions">
     <%= f.submit "Sign up" %>
   </div>
+  <%= link_to "¿have not you received the email with instruccions click here! ", new_user_confirmation_path %>
```

Comment:
- [ ] I have trouble loading up your sign-up page. I had to comment this line to ahead with the other requirements.

  ![image](https://github.com/Marcraphael12/rails-blog-app/assets/36542986/065e3648-7206-4bd6-b696-8c877cb5464b)